frontend/src/forms/subforms/LandForm.tsx

```diff
@@ -117,6 +117,20 @@ const LandForm = <T extends any>(props: LandProps & FormikProps<T>) => {
               field={withNameSpace('agency')}
             />
           </Form.Row>
+          {getIn(props.values, withNameSpace('subAgency')) && (
+            <Form.Row>
+              <Form.Label column md={2}>
+                Sub-Agency
+              </Form.Label>
+              <FastInput
+                formikProps={props}
+                disabled={true}
+                type="string"
```

Comment:
If not too much trouble I'd prefer this be named "data-type" or something like that to avoid confusion with `<input type="text">`

What do you think?